### PR TITLE
Update performance-questions.md

### PR DESCRIPTION
**Pull request by nsinghal12**
This PR is for a question to talk about optimizing resources JS/CSS/Images for performance.

* Breaking PR into separate PRs

---

Adding 3 new questions on performance.

### CODE REVIEW

File: questions/performance-questions.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 * What tools would you use to find a performance bug in your code?
 * What are some ways you may improve your website's scrolling performance?
 * Explain the difference between layout, painting and compositing.
+* What are different ways/tools to optimize resources being served with a page?
```
Comment (vvscode) [inline questions/performance-questions.md]: Please - create separate PR's per each question

File: questions/performance-questions.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+* How does CDN help in improving website performance?
```
Comment (vvscode) [inline questions/performance-questions.md]: looks like it's more for `networks` section

File: questions/performance-questions.md (hunk 1 above)
Comment (nsinghal12) [reply questions/performance-questions.md]: Sure.

**Comment by vvscode** [comment]
@nsinghal12 a few more conflicts

**Pull request closed without merging**